src/elements/Segment/SegmentGroup.js

```diff
@@ -0,0 +1,42 @@
+import cx from 'classnames'
+import React, { PropTypes } from 'react'
+import META from '../../utils/Meta'
+import {
+  getUnhandledProps,
+} from '../../utils/propUtils'
+import _Segment from './_Segment'
+
+function SegmentGroup(props) {
+  const {
+    children, className,
+  } = props
+
+  const classes = cx(
+    className
+  )
+
+  const rest = getUnhandledProps(SegmentGroup, props)
+
+  return (
+    <_Segment className={classes} {...rest} _sdClass='sd-segment-group' _segmentClass='segments'>
+      {children}
+    </_Segment>
```

Comment:
I had noticed how the docs are generated, but wasn't sure how private class documentation should be handled.  In the instance of Segment, which is basically a wrapper class for the logic laden _Segment, non of _Segments documentation has any method of being exposed for the root class Segment.

The re-structuring you suggested above, has solved this in the case, but doesn't for other similarly nested components.    What do you think the best way of remedying this is ?